client/app/routes/index.coffee

```diff
@@ -29,11 +29,9 @@ module.exports = class Router extends Backbone.Router
 
         @listenTo app.model, 'change:dialog', (model, id) ->
             return if id
-            
-            path = if _ref = app.model.get 'filter' then "tags/#{_ref}"
-            else 'contacts'
-            @navigate path, trigger: true
 
+            filter = (app.model.get('filter') or '').match /tag:(\w+)/i
```

Comment:
Of course yes, but when I tried, it failed. Recent test makes it working like a charm, let's switch the syntax :smile:.